src/java.base/share/classes/sun/security/validator/EndEntityChecker.java

```diff
@@ -356,12 +356,6 @@ private void checkCodeSigning(X509Certificate cert, Set<String> exts)
      */
     private void checkTSAServer(X509Certificate cert, Set<String> exts)
             throws CertificateException {
-        if (checkKeyUsage(cert, KU_SIGNATURE) == false) {
-            throw new ValidatorException
-                ("KeyUsage does not allow digital signatures",
-                ValidatorException.T_EE_EXTENSIONS, cert);
-        }
-
```

Comment:
Could you add a comment here just in case someone else adding the checking back?  Otherwise, looks nice to me.